src/styles/_mixins.scss

```
@@ -0,0 +1,21 @@
+@mixin mq($mq-breakpoint, $mq-breakpoints) {
```

@import 'vars';

```suggestion
@mixin mq($mq-breakpoint, $mq-breakpoints: $breakpoints) {
```
Dzięki temu mógłbyś `@include`ować ten `@mixin` tylko z 1 parametrem :)